src/rd.h

```diff
@@ -446,3 +447,5 @@ typedef struct rd_chariov_s {
         char  *ptr;
         size_t size;
 } rd_chariov_t;
+
+#endif /* NDef RD_H */
```

Comment:
This is a true nit, but I dont like "NDef", it made me wonder what it was for 2 seconds before I realized.
so to avoid that grave confusion can we go with just `#endif /* !_RD_H */` (including the prefixing underscore)?